source/MaterialXView/Material.h

```diff
@@ -70,6 +70,19 @@ class Material
         _elem = val;
     }
 
+    /// Return the material element associated with this material
+    mx::TypedElementPtr getMaterial() const
```

Comment:
Cache the MTLX material on the Viewer "material" class create on load. For usage later usage. Unlike shaderref a shader node can be used by > 1 material node so cache the one associated with the shader node here.

Reply:
Nitpick: Maybe rename this to `getElement()` or `getMaterialElement()`. I was confused by seeing `material->getMaterial()` in the code.